frontends/web/src/routes/account/send/send.tsx

```diff
@@ -46,10 +47,11 @@ import { FeeTargets } from './feetargets';
 import style from './send.module.css';
 import { TSelectedUTXOs, UTXOs } from './utxos';
 import { route } from '../../../utils/route';
+import { UnsubscribeList, unsubscribe } from '../../../utils/subscriptions';
 
 interface SendProps {
     accounts: accountApi.IAccount[];
-    code?: string;
+    code: string;
```

Comment:
changed to making code prop mandatory, so we don't have to do `if (this.props.code) { ` in the subscription callback